hr_timesheet_sheet/tests/__init__.py

```diff
@@ -1,3 +1,3 @@
 # License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
 
-from . import test_hr_timesheet_sheet
+from . import test_hr_timesheet_sheet_
```

Comment:
Why changing this?

Reply:
To keep blame as before I use this flow:

`cp file new; mv file new2; commit; upd files; commit`

Then I need to rename initial file. If not blame is moved to my name

Reply:
No, that's not something you can control. Try to blame on your fork and you'll see. One of the 2 parts won't have knowledge of this "split". I also prefer to keep them together. The `common.py` file is not following the naming convention (using the class name as the file name), and may be a disaster box for a lot of classes.

Reply:
`test_hr_timesheet_sheet_common`?

Reply:
@pedrobaeza FYI it can be controlled yes, look here

https://github.com/OCA/timesheet/blame/d692b6832d0fa4f23db2891d9b4ce8f5065ad2be/hr_timesheet_sheet/tests/common.py
https://github.com/OCA/timesheet/blame/d692b6832d0fa4f23db2891d9b4ce8f5065ad2be/hr_timesheet_sheet/tests/test_hr_timesheet_sheet_.py

But i will update only 1 file in future commit

Reply:
Good that this works. I think that when more and more the diff diverges, Git/GitHub won't be able to distinguish the split. I have already found it when blaming long history files.